modules/programs/git.nix

```diff
@@ -753,6 +753,54 @@ in
               };
           in
           lib.attrsets.mapAttrs' toSystemdTimer cfg.maintenance.timers;
+
+        launchd.agents =
+          let
+            baseArguments = [
+              "${cfg.package}/bin/git"
+              "for-each-repo"
+              "--keep-going"
+              "--config=maintenance.repo"
+              "maintenance"
+              "run"
+            ];
+          in
+          {
+            "git-maintenance-hourly" = {
+              enable = true;
+              config = {
+                ProgramArguments = baseArguments ++ [ "--schedule=hourly" ];
+                StartCalendarInterval = map (hour: {
+                  Hour = hour;
+                  Minute = 53;
```

Comment:
What's the significance of the `Minute = 53` in all of these units?

Reply:
i was just matching the times above

  https://github.com/nix-community/home-manager/blob/ddc65de72b8d46bdd63bb2da28697dfe9e34c1eb/modules/programs/git.nix#L282

[git picks a random minute when generating maintenance jobs to avoid everyone fetching at the same time](https://github.com/git/git/blob/master/builtin/gc.c#L1990). i don't think there's a super easy way to do that from nix? i could be wrong.